src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -120,20 +120,33 @@ public PublicClientApplicationBuilder WithIosKeychainSecurityGroup(string keycha
         }
 
         /// <summary>
-        /// On Android and iOS, brokers enable Single-Sign-On, device identification,
+        /// Brokers enable Single-Sign-On, device identification,
         /// and application identification verification. To enable one of these features,
         /// you need to set the WithBroker() parameters to true. See https://aka.ms/msal-net-brokers 
         /// for more information on platform specific settings required to enable the broker.
+        /// 
+        /// On iOS and Android, Autheticator and Company Portal serve as brokers.
+        /// On Windows, WAM (Windows Account Manager) serves as broker. See https://aka.ms/msal-net-wam
         /// </summary>
         /// <param name="enableBroker">Determines whether or not to use broker with the default set to true.</param>
         /// <returns>A <see cref="PublicClientApplicationBuilder"/> from which to set more
         /// parameters, and to create a public client application instance</returns>
-#if !SUPPORTS_BROKER
-        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]  // hide confidential client on mobile
-#endif
         public PublicClientApplicationBuilder WithBroker(bool enableBroker = true)
         {
-#if DESKTOP || WINDOWS_APP || NET5_WIN
+#if NET45
+            throw new PlatformNotSupportedException("The Windows broker is not available on .NET Framework 4.5, please use at least .NET Framework 4.6.2");
+#endif
+
+#if NET461 || NET_CORE
```

Comment:
Shouldn't this be specified with all versions of Windows Desktop broker? Same flag as below?

Reply:
The main difference between NET fwk and NET Core is the WinForms dependency, which defines the type `IWin32Window`. That's why all these #if flags are needed.